modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa News post updates.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\field\Entity\FieldConfig;
+
+/**
+ * Update body and summary labels.
+ */
+function oe_content_news_post_update_field_labels(array &$sandbox): void {
```

Comment:
We have agreed that new upgrade path names has to include number from now on! Please fix all.